### DISCUSSION TITLE
External links option in Android app

### DISCUSSION THREAD

**Discussion by manojm97** [community/community#11450] [Mobile]
Hi Team,

Please add external link option in settings to Android app similar to iOS app.

It will be very useful for so many people.

**Comment by stevepopovich** [comment]
Hi Manoj, can you clarify exactly what you are looking for? Can you screen record what you see in the iOS app that what you would want to see in the android app?

**Reply by manojm97** [reply]
Hi Steven,

Thanks for your response.

I mean to say that the external links option in app settings is available on iOS app but it is not there in Android app to open an external links inside github app (which would be very useful) instead of going to browser. 

Attaching the screenshots of both iOS and Android for your reference.

![External links options in iOS app](https://user-images.githubusercontent.com/61584819/156935945-be46eac4-ba16-4c80-a550-12a0c70ff28b.png)

![iOS app settings which has external links option](https://user-images.githubusercontent.com/61584819/156935970-487fe39a-f0dc-4619-8d64-8a9a44f69cee.png)

![Android app settings which doesn't have external links option](https://user-images.githubusercontent.com/61584819/156936006-dcc58753-faae-4b18-b270-ed6df67b5c45.jpeg)

Regards,
Manoj

**Reply by stevepopovich** [reply]
Hey Manoj. So, the Android app supports deep linking into the app for a subset of github.com links. We don't have a setting for it however. But clicking links like:
https://github.com/facebook/react
https://github.com/facebook/react/issues/24032
https://github.com/facebook/react/pull/24031

Should open in the app. For features we have not built in the mobile, we open to the browser. Links like:
https://github.com/facebook/react/pulse
https://github.com/facebook/react/actions

Should option in the browser.

Does that make sense?

**Reply by manojm97** [reply]
Hi Steven,

Yes I understood. 

Thanks for your response with an example.

It would have been even better if we had an option to select between opening any links In-App or directly via browser how we do it in iOS app.

But anyways I understand how it works in Android and thanks for your prompt response with the details.

Regards,
Manoj M
